src/create-envfile.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/python
```

Comment:
If we're going to be using a shebang, this shouldn't be able to call Python 2 right? I know it's in a Docker image, so I assume that it's covered, but I just wanted to make note of it.

Reply:
In container from image python:3.9 or python:10 or python:latest, path - /usr/bin/python leads to the version specified in the container.

Reply:
Oops, this path undefined. maybe I will point out /usr/bin/python3.9?